maintenance/measure_tile_performance/README.md

```diff
@@ -0,0 +1,83 @@
+# Measure Tile Performance
+
+This Measure Tile Performance script, `measure_tile_performance.py` will output the runtime results of running a given input query (based on a Looker Explore slug) a given number of times.
+
+It runs by invoking the Looker API, which requires API Keys to access which must be set in the `config/looker.ini` file.
+
+The script takes positional command line arguments to set the Explore slug and the number of times you want to run the query.
+
+The output on the command line will report the duration of each query as a list and the _min_, _max_, and _average_ query runtimes of those queries. It will optionally output a csv file with the headers:
+
+```
+SlugID, Timestamp, RunTime
+```
+
+Format:
+ - `SlugID`: The slugID that was provided as the first argument when running the script (see "Running" section below).
+ - `Timestamp`: The ISO 8601 Looker system time timestamp when the query was performed, such as: `2022-01-25T20:20:18Z`. The GDX Analytics Looker system timezone is set to UTC.
```

Comment:
@veenupunyani : It's probably easier if we do this in Pacific Time. Otherwise team members will likely have to do the mental math every time they run a report 

cc @mark-walle

Reply:
@danpollock seen. I'll make this change and commit the updated code to accomodate. It's not completely trivial, especially for accounting for DST offsets, so I'll handle it in order to get this PR approved.

Reply:
We can use Python native timezone handling. 

Something like this shows one easy way: 

```
import os
import time

os.environ['TZ'] = 'America/Vancouver'
time.tzset()
print ( time.strftime('%Y-%m-%dT%H:%M:%S%z'))
os.environ['TZ'] = 'America/Toronto'
time.tzset()
print ( time.strftime('%Y-%m-%dT%H:%M:%S%z'))
```

Reply:
Too late, I've already used pytz.

pytz localizes and adjust tz without requiring environment changes, which would later need to be reverted when writing the filename again.

Reply:
Thanks @mark-walle I have tested and it works just fine. Resolving this comment now